lms/envs/common.py

```diff
@@ -1247,7 +1247,8 @@
     'js/vendor/requirejs/require.js',
     'js/RequireJS-namespace-undefine.js',
     'js/vendor/URI.min.js',
-    'js/vendor/backbone-min.js'
+    'js/vendor/backbone-min.js',
+    'js/vendor/afontgarde/modernizr.fontface-generatedcontent.js',
```

Comment:
Why is it necessary to put this modernizr library in base_vendor? When will it be used?

We generally want to load as little as possible in base_vendor.